projects/go-lib/src/lib/components/go-table/go-table.component.ts

```diff
@@ -11,6 +19,9 @@ import { sortBy } from './go-table-utils';
 export class GoTableComponent implements OnInit, OnChanges {
 
   @Input() tableConfig: GoTableConfig;
+  @Input() loadingData: boolean = false;
```

Comment:
Why are we allowing this to be an input?

Reply:
@akfernun so that we can control the loading state dependent upon server requests.